aiohomekit/model/services/service.py

```diff
@@ -41,7 +41,10 @@ def filter(self, char_types=None) -> Iterable[Characteristic]:
         matches = iter(self)
 
         if char_types:
-            char_types = [CharacteristicsTypes.get_uuid(c) for c in char_types]
+            char_types = [
+                (c if len(c) == 36 else CharacteristicsTypes.get_uuid(c))
```

Comment:
Can we do this normalisation in get_uuid?

Reply:
Yes, I originally had it there. I will add it back and update the tests.

Reply:
Brilliant, thank you. When that's done I can push a tag/pypi release with it merged for you if you'd like?